.github/workflows/update.yml

```diff
@@ -0,0 +1,33 @@
+name: Update
+on:
+  pull_request:
+    types: closed
+  workflow_dispatch:
+jobs:
+  build:
+    name: Deploy docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout LFS objects
+        run: git lfs pull
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 'pypy-2.7'
+
+      - name: Build pages
+        uses: Exabyte-io/mkdocs-s3-publish-action@main
```

Comment:
Per our discussion, let's fork the repos as:
1. Exabyte-io/action-mkdocs-s3-publish@main
2. Exabyte-io/action-install-aws-cli@main

Reply:
done.

Reply:
`Exabyte-io/mkdocs-s3-publish-action@main` should be `Exabyte-io/action-mkdocs-s3-publish@main`